templates/single-quiz.php

```diff
@@ -57,7 +57,10 @@
 					sensei_setup_the_question();
 					?>
 
-					<li class="<?php sensei_the_question_class(); ?>">
+					<li
+						class="<?php sensei_the_question_class(); ?>"
+						value="<?php echo esc_attr( sensei_get_the_question_number() ); ?>"
```

Comment:
Wow, I've never seen this attribute used on a list item before. 🆒 

Since we've changed a template file and some people override them with their own versions, we'll need to bump the template version, add the "Templates" label and make note of which template was changed in the PR description.

Reply:
> Wow, I've never seen this attribute used on a list item before.

Same... 😄 

>Since we've changed a template file and some people override them with their own versions, we'll need to bump the template version, add the "Templates" label and make note of which template was changed in the PR description.

Version bumped in d50a5e259f3da2446db04581920d555c6d28bf53, thanks!